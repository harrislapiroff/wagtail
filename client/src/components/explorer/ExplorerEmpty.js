import React from 'react';
import { STRINGS } from '../../config/wagtail';

const ExplorerEmpty = () => (
  <div className="c-explorer__placeholder">{STRINGS.NO_RESULTS}</div>
);

export default ExplorerEmpty;
