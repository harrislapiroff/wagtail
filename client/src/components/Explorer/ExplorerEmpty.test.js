import React from 'react';
import { shallow } from 'enzyme';

import ExplorerEmpty from './ExplorerEmpty';

describe('ExplorerEmpty', () => {
  it('exists', () => {
    expect(ExplorerEmpty).toBeDefined();
  });

  it('basic', () => {
    expect(shallow(<ExplorerEmpty />)).toMatchSnapshot();
  });
});
