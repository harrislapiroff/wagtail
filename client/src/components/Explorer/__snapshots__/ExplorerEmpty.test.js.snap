exports[`ExplorerEmpty basic 1`] = `
<div
  className="c-explorer__placeholder">
  No results
</div>
`;
